x-pack/plugins/alerting/server/saved_objects/schemas/raw_rule/v1.ts

```diff
@@ -209,6 +210,7 @@ const rawRuleActionSchema = schema.object({
     })
   ),
   alertsFilter: schema.maybe(rawRuleAlertsFilterSchema),
+  useAlertDataForTemplate: schema.maybe(schema.boolean()),
```

Comment:
This is/was a new field and causes a validation error.

Reply:
I don't understand the purpose of `field` and `useAlertDataForTemplate`.  Is for use in the future?  It seems like it could have been a test you were running locally, didn't intend to commit, but you added a comment, so seems not.

Reply:
Someone added that new field and forgot to update the rawRuleSchema, therefore createRule method fails.
I added it to fix the problem.

Reply:
I don't know how they missed that, task execution would be skipped when that field is in any rule.

Reply:
Interesting!  I wonder if it coincided with some other "leniency" PRs where we might have been more lenient in accepting some objects validating schemas (allowing extra fields, but ignoring).  

I was going to suggest opening an issue to figure out how this happened, because it doesn't seem good.  However, we ARE now catching it :-), so ... not sure it matters.  Presumably we'd catch the next time this happens.

Reply:
Actually that's why we decided to use modelVersions. It will force developers to bump the version.
No overlooked new fields anymore :)